src/primitives/polyline/styled.rs

```diff
@@ -246,6 +296,51 @@ mod tests {
         );
     }
 
+    #[test]
+    fn thick_stroke_translated() {
+        let mut display = MockDisplay::new();
+
+        let styled = Polyline::new(&PATTERN)
+            .translate(Point::new(-4, -3))
+            .into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 4));
+
+        assert_eq!(
+            styled.bounding_box(),
+            Rectangle::new(Point::zero(), Size::new(28, 10))
+        );
+
+        styled.draw(&mut display).unwrap();
+
+        display.assert_pattern(&[
+            "         #                  ",
+            "       #####            ##  ",
+            "      #######         ##### ",
+            "    ##########      ####### ",
+            "   #############  ##########",
+            " ######## ################# ",
+            "#######     #############   ",
+            " ####         #########     ",
+            "  #            ######       ",
+            "                 ##         ",
+        ]);
+    }
+
+    #[test]
+    fn thick_stroke_points() {
+        let mut d1 = MockDisplay::new();
+        let mut d2 = MockDisplay::new();
+
+        let pl = Polyline::new(&PATTERN)
+            .translate(Point::new(2, 3))
+            .into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 4));
+
+        pl.draw(&mut d1).unwrap();
+
+        pl.into_pixels().draw(&mut d2).unwrap();
+
+        assert_eq!(d1, d2);
```

Comment:
```suggestion
        d1.assert_eq(d2);
```

Reply:
Alright alrightt! You write some cool code I gotta use it!

Joke ;)